### PR TITLE
refactoring regexp-pick

### PR DESCRIPTION
**Pull request by stoeffel**
In my opinion this is much simpler to read.

refs tjmehta/101/pull/55

### CODE REVIEW

File: pick.js [hunk 1]
```diff
@@ -27,49 +27,30 @@ module.exports = function () {
...
 
-function copyWithRegExp (from, to, regexp) {
-  return function (key) {
-    if (regexp.test(key)) {
-      to[key] = from[key];
-    }
-  };
-}
\ No newline at end of file
+function isRegExp (obj) {
```
Comment (stoeffel) [inline pick.js:54]: This could even be a seperat module.
@tjmehta what do you think?

Comment (tristaaan) [inline pick.js:54]: I was wondering this too.

Out of curiosity, what is the difference between this and `obj instanceof RegExp`?

Comment (stoeffel) [inline pick.js:54]: In this case using `instanceof` would be okay. It's only a problem if you have different contexts(windows). See i.e. https://github.com/strongloop/express/issues/1557

Comment (tristaaan) [inline pick.js:54]: Thanks for the `instanceof` explanation and the refactor, this is _much_ easier to read. Something I need to improve on.

Comment (stoeffel) [inline pick.js:54]: No problem :bowtie:

**Pull request merged**